doc/lsst.meas.extensions.scarlet/overview.rst

```diff
@@ -0,0 +1,217 @@
+
+*scarlet-lite* was designed specifically for use in the science pipelines and is therefor able to run in a smaller memory footprint in a fraction of the run time of scarlet main.
+In this case the data is assumed to be organized as if it was taken from a hyper-spectral camera, meaning the observation in each band is covering the same spatial area, in the same orientation, on the same pixel grid.
+This allows for a much simpler class hierarchy and gradients that can be calculated analytically, avoiding the use of an external automatic differentiation package and the memory and runtime overhead that creates.
+
+.. note::
+    The *scarlet* developers are currently working on porting from ``autograd`` to either `JAX <https://github.com/google/jax>`_ or `pytorch <https://github.com/pytorch/pytorch>`_.
+    In either case, the general performance issues present in ``autograd`` will still be present.
+
+However, the underlying models and algorithms in scarlet and scarlet-lite are *identical*, the only difference between them is the architecture of the code used to generate and optimize the models.
+For this reason the remainder of this document will refer to the multi-band deblending algorithm as *scarlet*, even though in practice the actual package implementing the algorithm is *scarlet lite*.
+
+.. _basic_scarlet_model:
+
+Basic Model
+^^^^^^^^^^^
+
+Just like the single-band deblender, in order to make a model for a multi-band image we will have to make some assumptions.
+In this case the main assumption is that an astrophysical source can be modeled as a collection of components, where each component can be described by a single SED that is roughly constant but can vary in magnitude over its spatial extent (morphology).
+This assumption can be justified physically by considering the structure of a galaxy.
+In general the populations of stars in the bulge and disk are different, as well as perhaps additional populations of star forming regions, each with its own distinct SED.
+However, the actual colors in each population are not that different, which is why parametric bulge-disk models (Sersic models, Spergel models, Gaussian mixture models, etc.) have been successfully used in deblending and photometric measurement.
+
+The other assumption that is currently made in the science pipelines models is that each component is monotonically decreasing from a peak.
+Again, thinking of galactic structure this is usually a reasonable assumption as both the bulge and disk are brightest in the center, and even spiral arms generally emanate from a peak location.
+There are exceptions, however, like nebula and irregular galaxies that are much less organized and not well constrained by a component-like model with a center.
```

Comment:
Updated the typo.
Symmetry is no longer used, so galaxies do not have to be symmetric anymore, but it must be possible to decompose them into a small number of monotonic components. Gaseous regions like nebulae and irregulars break that assumptions because there is no central monotonic region, so even their "center" is ill-defined in terms of where to put the scarlet model.